src/styles/headings/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Headings
+description: Use headings to structure content on pages
+section: Styles
+theme: Typography
+layout: layout-pane.njk
+showPageNav: true
+order: 7
+---
+
+{% from "_example.njk" import example %}
+
+Write all headings in sentence case.
+
+Use heading tags, such as `<h1>`, `<h2>` and so on, to tag the headings on a page. Apply a heading class, such as `govuk-heading-l`, to style them visually. Style headings consistently to create a clear content structure throughout your service.
+
+For a [question page](/patterns/question-pages/), or pages with long headings, start with `govuk-heading-l` for an `<h1>`, `govuk-heading-m` for an `<h2>` and so on. But change it if your pages feel unbalanced – the heading class you use does not always need to correspond to the heading level.
+
+{{ example({ group: "styles", item: "headings", example: "default", html: true, open: true, size: "m", loading: "eager" }) }}
+
+If your page has lots of long form content, start with `govuk-heading-xl` for an `<h1>`, `govuk-heading-l` for an `<h2>`, and so on.
+
+{{ example({ group: "styles", item: "headings", example: "headings-xl", html: true, open: true, size: "m" }) }}
```

Comment:
Should we remove govuk-heading-s from that example? It's just another h3, however I wonder if it might mislead users into thinking that h4s are fine in this instance.

Reply:
Remove the last example for now
Look into existing guidance on number of heading levels for long form content